src/lib/wallet/GoodWalletProvider.js

```diff
@@ -56,7 +63,12 @@ export const GoodWalletProvider = ({ children, disableLoginAndWatch = false }) =
 
   const update = useCallback(
     async goodWallet => {
-      const { tokenContract, UBIContract, identityContract, account } = goodWallet
+      const { tokenContract, UBIContract, identityContract, account, networkId } = goodWallet
+
+      if (!supportsG$(networkId)) {
```

Comment:
still need to update default zero/false values for networks where ubi/G$ is not available.
again G$ token is available on mainnet and UBI isnt. so support for token needs to be separated from support for UBI